recipes/mesalib/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "MesaLib" %}
+{% set version = "7.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.freedesktop.org/pub/mesa/older-versions/7.x/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2b5595952e94d5ecdb7c0a53ac6db9bbf23cc2228b09124ae0c37f9507690ac
+
+build:
+  skip: True  # [win or py3k]
+  number: 0
+  track_features:
+    - mesalib
```

Comment:
I am not sure exactly what you are expecting here @doutriaux1. @msarahan what do we need to do to track custom features?